models/klaviyo__flows.sql

```diff
@@ -0,0 +1,37 @@
+with flow as (
+
+    select *
+    from {{ var('flow') }}
+),
+
+flow_metrics as (
+
+    select *
+    from {{ ref('int_klaviyo__campaign_flow_metrics') }}
+    where last_touch_flow_id is not null
+),
+
+flow_join as (
+    
+    {% set exclude_fields = [ 'last_touch_campaign_id', 'last_touch_flow_id'] %}
```

Comment:
similar to my previous comment -- would love to hear your thoughts on the cost/benefit of including this.